src/cljs/rems/application.cljs

```diff
@@ -461,8 +461,8 @@
        [:h3 (cond (= (:application/applicant application) user-id) (text :t.applicant-info/applicant)
                   (:userid attributes) (text :t.applicant-info/member)
                   :else (text :t.applicant-info/invited-member))]
-       (when-let [name (or (:commonName attributes) (:name attributes))]
-         [info-field (text :t.applicant-info/name) name {:inline? true}])
+;;       (when-let [applicant-name (get-applicant-name application)]
+;;         [info-field (text :t.applicant-info/name) applicant-name {:inline? true}])
```

Comment:
Commented left overs or accidental? get-applicant-name is not used elsewhere.

Reply:
Done. The lines had been accidentally commented.